plugins/services/src/js/utils/PodUtil.js

```diff
@@ -90,7 +90,7 @@ var PodUtil = {
       // Filter combined container list to remove potential duplicates
       const containerIds = new Map();
       combinedContainers = combinedContainers.filter(function({ containerId }) {
-        if (!containerIds.has(containerId)) {
+        if (!containerIds.has(containerId) && containerId != null) {
```

Comment:
What do you think about the null check being first? We could have one less check.